samples/subsys/llext/modules/sample.yaml

```diff
@@ -4,24 +4,38 @@ common:
     - arm
     - xtensa
   platform_exclude:
+    # platforms with active issues
     - apollo4p_evb            # See #73443
     - apollo4p_blue_kxr_evb   # See #73443
     - numaker_pfm_m487        # See #63167
+    - s32z2xxdc2/s32z270/rtu0 # See commit 18a0660
+    - s32z2xxdc2/s32z270/rtu1 # See commit 18a0660
+    # platforms that are always skipped by the runtime filter
+    - qemu_arc/qemu_arc_em
+    - qemu_arc/qemu_arc_hs
+    - qemu_arc/qemu_arc_hs/xip
+    - qemu_arc/qemu_arc_hs5x
+    - qemu_arc/qemu_arc_hs6x
+    - qemu_cortex_m0
+    - qemu_xtensa/dc233c/mmu
   integration_platforms:
```

Comment:
Why remove qemu_xtensa

Reply:
The MMU-enabled Xtensa is not compatible with llext at the moment: there is no `writable_mpu` test, as it would require to split up the ELF file in mem_domains (with all the relevant alignment issues...). 
I could leave the platform in, but it would re-do the same tests of the non-MMU `qemu_xtensa` variant.